src/app/zap-templates/zcl/zcl.json

```diff
@@ -30,7 +30,7 @@
         "sample-extensions.xml"
     ],
 
-    "manufacturersXml": "../shared/manufacturers.xml",
+    "manufacturersXml": "../../../../third_party/zap/repo/zcl-builtin/shared/manufacturers.xml",
```

Comment:
could we instead import the file into CHIP?

I think we are working towards not having source dependency on zap.

@jepenven-silabs ?

Reply:
Yes we should move this file within CHIP repo as to no have any ZCL dependency with ZAP. @markus-becker-tridonic-com always assume that if the files within ZAP `zcl-builtin` folder might change at any moment.

Reply:
@markus-becker-tridonic-com could you copy the `manufacturers.xml` inside the `/src/app/zap-templates/zcl` folder and update the path ?